### DISCUSSION TITLE
Provide the Ability to Merge NFT Plots from different Launcher IDs

### DISCUSSION THREAD

**Discussion by vdx660** [Chia-Network/chia-blockchain#8062] [Ideas]
There's currently no ability to merge NFT Plot created from different Launcher IDs. This means that if you are currently farming in 2 Pools and would want to Change Plot to farm only one 1 Pool, you will get 2 Launcher IDs farming on the same Pool.

**Comment by keliew** [comment]
https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-FAQ#what-is-a-plot-nft

Not possible...yet?

**Comment by BrandtH22** [comment]
@vdx660 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - since the plot themselves are created using the plot NFT contract address there is no method to combine or merge these plotNFTs. Furthermore there is no current method to delete plotNFTs from the client. If you do want all of your plots under a single plotNFT you will need to delete any plots created with the second plotNFT (or OG plots) and replot them using the desired plotNFT.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
